Notebooks/DC2 Coadd Run1.1p GCR access -- Tutorial (part1).ipynb

```diff
@@ -0,0 +1,1401 @@
XGAj8d5755y39jnn7gZwNwCsRHEuo1oRKUrLWZaObklzVgkzK3+eLsdlmRssI4dVTg4lOQqtvHhiOLBeuLckPMvkoct+WYaImRJpVbBMM5lzN0rRyfXUxoLt9z0T12Ndz4qxAEyp05Lfhz+WvDA99GRn7hhWibzMeRml/ywZWB/PMkRYcrcl2XdYRiGVR7Ao/54ea+UbtI6nZewxw3SSltUzXjmwSgmWYcrhTc5S37/IpWEhUqZlkvAT0Wzhd18FAHCHlHQaZFYp/QZEs8jkz/1U2iZl4mZ7opB4bmvyCDG7Mv77ZHbLVLp9ed8YAGD/uvzXVptFRBqeWxkfSbxROk7MMJb0a7UBSyMJt7o5pZYHQfl/1AcA3OK9nwUA59z/gOQdl19q0FpGAFyh/n55aKulz4qCscedc1u8968HGeaEdXDv/UMAHgKA1c741hJCSHV4/yKEtCS1vCP4mnPuLwBsApCGTrz3fwfgygau5XkAVzvnrnTOdQP4CIC9FX32Avh4cN+9B8B4kE2Kxu4FcFfYvgvA1xu4ZkIIAXj/IoS0KLVEBMcAPIXkX5t/7Zx7AsB+ANejuqI1b7z3M865TwP4BoBOAF/23u93zt0T9n8JwONIIpPDACYA/GrR2DD1AwAedc79OoBXAdzZqDWXIfKoJVuWYY2xJDDdJlKhltlEYi5yrAK2jKjnPr85+TdD/6NR9usLc4rkCQDXfDJ/flZZNrk2Vu45QJU902XbDFlTy6PCqmN5GTuVK5UsKcfQcrDGyssokqheg+TA2/BiDMT0GZ+PXBvL/W19jkC21FsluuyczKml9AGj7Jxc95Eg2QLx+6LLEO48Gtc4ucnn1iglB3U+QuuVA5GTrdcVLIlYr9H6zp56R3QXDzyXH2uV7Kt2nNy+V6p2K2U53r/IpaFIWizLn2fm0gul4QAlx/apfIQB7TiWl1e6x9fE/SvzjwsXL0t+j+teisf1U/FOf3QieZlmxbn4uxVHsh+4LB7vQvJ/qo4L6qWo3dcmx90fy+HJGvV1kNyKnSW5EReTVpSDNbU8CF7vvf8159wLSN4R/CSAXwFwBsD/0sjFeO8fR3Kz1G1fUtsewKdqHRvaTwO4tZHrJISQSnj/IoS0IrU8CD7pnHsGiVv4fwbwbQAPeu+niocRK/KTRm5UNE62yyKHus3KXZe+kG/lG1SRG6uqh5XXUL94L9EnvUYJB1/zyefTNit/XrdhppD1Z4wMJVUjxKihI2YScdI57iqPq89FR7AkyjlQxShjRWLF3KLXtXNP9bHa9GOZeazolzZWWBHIyvUDKmqpo5dD+ejBqRD12/ZENH7IsXVEU1/j7fclEcMxIw9fn1q3lfNR2nQVEctoYs2jo5vynd72RHzx3bo2RZFtKzI4YVwjQpqVssiTZVroOPhq2jZzQxJl69wXo2xiFukMEbhqrBhP5hu/fnXatuU7icHk/LZog+w4H6N/u1YneQtfefPqtE0igTNr4lo7pgx75Gj+dVg5v0yFFZl3cjLXD2j9aN1SUPog6L3/HefcVQD+Ack7gR8EsMs5dxHAj7z3DY0KEkIIIYSQpaGmyiLe+4POufd5738sbc65fgBvX7SVEUIIIYSQRaXmEnP6ITD8/RyAvJ5IUmqVfC25uMt42V3LrUXzWGjp18w3aBgLdH69SF7W09KqyHlWiTktHaayq5pH5xRM8xaq9YgUbUmnRSXwkrkTqUCbDUTy1KKElmgnDAlTzmVYGyv2jGTWAsT8jvozsSRRK8/eS5+NObl2PnKh6lid189C5FEtf4r8a0miWmId0DuMsnTWKwDWKwmpUUh99nIcLT8PPFecz2/EyGVYeYxqWGsomo+QZkPy55WZIDJl4kI5OV2OzQcJNtNP9ikJWUrQaclWcv1dWO9QSeeFaFzrGLyYbr90NjnOlPqJiiS84ljMW3j+2o0AgO5x5T9dk9xPul6OTXL+nYY0TAm4fuotMUcIIYQQQlocPggSQgghhLQpNUvDpH4ypccMyVCoJiGLfGqVndNYUrQ1t1UuTtaoJcPeE1ECsKRXmafMDWwh0qJORGk5PSVHHxAdo/p4MqaarF7ZbwPi+Yl8rd23OjefbhdEMs3kHjSOZ80t+fq0bD6+O5FRNu6L56RL0E0Mdebmkc9i2siJqPuJ7Fn2WoCFvp5WPj8rz6MlwVbu01ST863vsXXOlsRc9IqAPueyVwkIaSZqzYs3c+hwrk3fxSSP4FyQjasSHLuda2MeweM3J9srT8fsAuIW7j0e790z5+ILP5/Z+iQA4DfOR9dwV3Afn/rpLWlb97nkvntudZR81z6frHGmxnMX+bwScRNTOq4OI4KEEEIIIW0KHwQJIYQQQtoUSsMNptbScDpZbplrWNotJ65GZDNrbi2pWeXiRP6r5hwtWkNWZszLgzLWkhg1ulTdzkeSeXQCYSFzPcN5WS5rS6LURCnaTios0qqWy63rIPPYcngcazmE+0bzjnAruXK/+m7Id0gn0ZY5+0tKGMq10e5pcT3XWv4QsGVZuV6ZVwWMz6Ao2TagvifG56yRY+vraTnshb4SdzEhzU5ZomQtj4qcPLs5tvkXksqFOnm0lHDLlG0zStAN/lPi8n3lQzGh9IbvnwMQHcUA0Hsk3sf3TW4HAKw+PJubb83B6Ngfvyq5l234/17P9bPOWSePrjXJ9lInmW6lpNaMCBJCCCGEtCmMCDaYMtOCFamQiFq1aI7Mo6N1VpRN5i7KJwjYpegkijhdLWoXoik66iVr0JEWiVJtVEM7jPOSSJHOmWeZJPQ5S7RHR6GiuaN6xAgAJjclLzhnTB6B/lxLghXBlKiezAcAvZvz5e0kOqav1+it8i/luC4xi+w6qj4zI4qWMfuECKU2tljRvDmjrJ5czw0vxijutFGezvqe6uiefGd1Hj7r+2dFZyWaqqOSurydZQCStencljv35A0fHQV5LLUxSUxImYjzK7npCGkpdKRMooPuWDRbXPj5G5N9x8+l
```

Comment:
Please remove plots.  If you have an image you want to persist, make a PNG, and include in assets.